### PR TITLE
Add blenderseed shader folder

### PR DESCRIPTION
**Pull request by jdent02**
Add Closure2Surface blenderseed node

### CODE REVIEW

File: src/appleseed.shaders/src/blenderseed/as_closure2surface.osl [hunk 1]
```diff
@@ -0,0 +1,40 @@
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2018 Jon Dent
```
Comment (dictoon) [inline src/appleseed.shaders/src/blenderseed/as_closure2surface.osl]: This should be

    // Copyright (c) 2018 Jon Dent, The appleseedhq Organization

File: src/appleseed.shaders/src/blenderseed/as_closure2surface.osl [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	string as_maya_classification = "drawdb/shader/surface:rendernode/appleseed/surface:shader/surface:swatch/AppleseedRenderSwatch"
+]]
+(
+    closure color in_input = 0
+    [[
+    	string as_maya_label = "Input Closure"]]
+)
+{
+    Ci = in_input;
+}
```
Comment (dictoon) [inline src/appleseed.shaders/src/blenderseed/as_closure2surface.osl]: End file with a newline character (GitHub and many other tools complain about a file not ending on a newline).

File: src/appleseed.shaders/src/blenderseed/as_closure2surface.osl [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+surface as_closure2surface
+[[
+	string as_maya_node_name = "asClosure2Surface",
+	string as_maya_classification = "drawdb/shader/surface:rendernode/appleseed/surface:shader/surface:swatch/AppleseedRenderSwatch"
+]]
+(
+    closure color in_input = 0
+    [[
+    	string as_maya_label = "Input Closure"]]
```
Comment (dictoon) [inline src/appleseed.shaders/src/blenderseed/as_closure2surface.osl]: I believe the closing brackets should go on the next line.

**Comment by jdent02** [comment]
Updated

File: src/appleseed.shaders/src/blenderseed/as_closure2surface.osl [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+surface as_closure2surface
+[[
+	string as_maya_node_name = "asClosure2Surface",
```
Comment (est77) [inline src/appleseed.shaders/src/blenderseed/as_closure2surface.osl]: You can remove this line and the next one.

File: src/appleseed.shaders/src/blenderseed/as_closure2surface.osl [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+	string as_maya_classification = "drawdb/shader/surface:rendernode/appleseed/surface:shader/surface:swatch/AppleseedRenderSwatch"
+]]
+(
+    closure color in_input = 0
+    [[
+    	string as_maya_label = "Input Closure"
```
Comment (est77) [inline src/appleseed.shaders/src/blenderseed/as_closure2surface.osl]: You can also remove this one. We are not going to use this shader in Maya.

Comment (jdent02) [reply src/appleseed.shaders/src/blenderseed/as_closure2surface.osl]: The parser uses those for metadata.

Comment (est77) [reply src/appleseed.shaders/src/blenderseed/as_closure2surface.osl]: Maybe we should change them to as_blender_label and similar?

Comment (jdent02) [reply src/appleseed.shaders/src/blenderseed/as_closure2surface.osl]: Nah cause all the shaders in the appleseed folder use the as_maya nomenclature. No point adding more to the parsing code

**Pull request merged**